MainModule/Client/UI/Default/UserPanel.lua

```diff
@@ -1127,6 +1127,21 @@ return function(data)
 						toggle.Text = text
 					end
 				};
+				{
+					Text = "Hide Chat Commands: ";
+					Desc = "- Hide your commands run from the chat from others";
```

Comment:
Maybe reword for clarity?
```suggestion
					Desc = "- Hide commands ran from the chat";
```

Reply:
"Hide your chat commands from others" sounds better than both of them in my opinion

"Hide commands ran from the chat" implies that commands sent by others will also be hidden when that isn't the case
"Hide your commands run from the chat from others" is intricate but not necessarily unclear in my opinion